### PR TITLE
R doc 1911 global custom default analyzer

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown [hunk 1]
```diff
@@ -274,3 +274,33 @@ Smallest n-gram to generate when NGram analyzer is used.
 - **Scope**: Server-wide or per database
 
 {PANEL/}
+
+{PANEL:Indexing.Analyzers.Default}
+
+Default analyzer that will be used for fields.
+
+- **Type**: `string`
+- **Default**: `Constants.Documents.Indexing.Analyzers.Default`
```
Comment (ppekrol) [inline Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown:283]: Please write the value of a constant

File: Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown [hunk 2]
```diff
@@ -274,3 +274,33 @@ Smallest n-gram to generate when NGram analyzer is used.
...
+- **Scope**: Server-wide, or per database, or per index
+
+{PANEL/}
+
+{PANEL:Indexing.Analyzers.Exact.Default}
+
+Default analyzer that will be used for exact fields.
+
+- **Type**: `string`
+- **Default**: `Constants.Documents.Indexing.Analyzers.DefaultExact`
```
Comment (ppekrol) [inline Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown:293]: Please write the value of a constant

File: Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown [hunk 3]
```diff
@@ -274,3 +274,33 @@ Smallest n-gram to generate when NGram analyzer is used.
...
+- **Scope**: Server-wide, or per database, or per index
+
+{PANEL/}
+
+{PANEL:Indexing.Analyzers.Search.Default}
+
+Default analyzer that will be used for search fields.
+
+- **Type**: `string`
+- **Default**: `Constants.Documents.Indexing.Analyzers.DefaultSearch`
```
Comment (ppekrol) [inline Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown:303]: Please write the value of a constant

File: Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown]: Done

File: Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown]: Done

File: Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown (hunk 3 above)
Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/configuration/indexing-configuration.markdown]: Done

**Pull request merged**